action/tx_access_list.go

```diff
@@ -143,6 +146,8 @@ func fromAccessListProto(list []*iotextypes.AccessTuple) types.AccessList {
 			for j, key := range v.StorageKeys {
 				accessList[i].StorageKeys[j] = common.HexToHash(key)
 			}
+		} else {
+			accessList[i].StorageKeys = []common.Hash{}
```

Comment:
ethereum will marshall/unmarshall a `nil` slice value to `[]common.Hash{}`, so make it the same